resource/ignite2022-prod/session_map.json

```diff
@@ -1,64 +1,49 @@
 {
   "SessionSetMap": [
     {
-      "HubbId": 1008,
-      "SessionTypeName": "Featured Session",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
-    },
-    {
-      "HubbId": 1010,
-      "SessionTypeName": "Session",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
-    },
-    {
-      "HubbId": 1011,
+      "HubbId": 1,
       "SessionTypeName": "Keynote",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
-    },
-    {
-      "HubbId": 1012,
-      "SessionTypeName": "Learning Zone",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
-    },
-    {
-      "HubbId": 1013,
-      "SessionTypeName": "Connection Zone",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
+      "SessionSets": ["Happening Now", "Session Catalog", "Session Scheduler", "My Schedule"]
     },
     {
-      "HubbId": 1014,
+      "HubbId": 2,
       "SessionTypeName": "Studio",
       "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
     },
     {
-      "HubbId": 1015,
-      "SessionTypeName": "Always On",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
-    },
-    {
-      "HubbId": 1016,
-      "SessionTypeName": "Strategic Meeting",
-      "SessionSets": ["Session Catalog", "My Schedule", "Session Scheduler"]
+      "HubbId": 3,
+      "SessionTypeName": "Connection Zone",
+      "SessionSets": ["Happening Now", "Session Catalog", "Session Scheduler", "My Schedule"]
     },
     {
-      "HubbId": 1017,
+      "HubbId": 4,
       "SessionTypeName": "Breakout",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
+      "SessionSets": ["Happening Now", "Session Catalog", "Session Scheduler", "My Schedule"]
     },
     {
-      "HubbId": 1018,
+      "HubbId": 5,
       "SessionTypeName": "On-Demand",
       "SessionSets": ["Session Catalog", "Session Scheduler"]
     },
     {
-      "HubbId": 1019,
+      "HubbId": 6,
       "SessionTypeName": "Core Theme",
-      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
+      "SessionSets": ["Happening Now", "Session Catalog", "Session Scheduler", "My Schedule"]
+    },
+    {
+      "HubbId": 7,
+      "SessionTypeName": "Venue Videos",
```

Comment:
@jcollicoat What is the effect of not mapping these types? Seems weird to set the sessionSet to `[]` if it's not going to show up at all